.circleci/config.yml

```diff
@@ -1,6 +1,63 @@
 version: 2.1
 orbs:
   node: circleci/node@4.2.1
+commands:
+  set-environment:
+    parameters:
+      env:
+        default: "DEV"
+        type: "string"
+    steps:
+      - run: |
+        CF_SPACE=CF_SPACE_<<parameters.env>>
+        CF_USERNAME=CF_USERNAME_<<parameters.env>>
+        CF_PASSWORD=CF_PASSWORD_<<parameters.env>>
+        CGHOSTNAME_FRONTEND=CGHOSTNAME_FRONTEND_<<parameters.env>>
+        CGHOSTNAME_BACKEND=CGHOSTNAME_BACKEND_<<parameters.env>>
+  deploy-frontend:
+    parameters:
+      deploy-strategy:
+        default: "rolling"
+        type: string
+      deploy-env:
```

Comment:
This variable never actually gets used in either of the deploy scripts - I think it would be safe to remove.

Setting the space with the `cf target` command should suffice for what I think this was intended for.

Reply:
👍